src/operations/bits_manipulation/bits_manipulation.cairo

```diff
@@ -1,12 +1,12 @@
 use traits::Into;
-use pragma::operations::bits_manipulation::pow2::pow2;
+use pragma::operations::bits_manipulation::pow2::{pow2};
 
-fn actual_get_element_at(input: u256, at: u256, number_of_bits: u256) -> u256 {
+fn actual_get_element_at(input: u256, at: u256, number_of_bits: u256) -> u128 {
```

Comment:
shouldn't we change everything to u128 in this lib ?

Reply:
No, it's a bit trickier, if we change the configuration to u128, we will lack bits, since we are already using all the 128 bits(0-31 for timestamp, 32-62 for volume and 65-128 for price). So in order to store more value for volume either we can reduce the allocation for the price (in that case we can transform to u128) or keep the same allocation for the price ( in that case we are obliged to use u256)